### DISCUSSION TITLE
Can the same docs version support multiple routing path?

### DISCUSSION THREAD

**Discussion by casionone** [facebook/docusaurus#7502] [Q&A]
Can the same docs version support multiple routing path
```
versions: {
            current: {
              path: '1.1.2',
            },
            '1.1.1': {
              path: 'latest',
            },
          }

```
The current latest version is 1.1.1 (latest), which can be accessed through https://linkis.apache.org/docs/latest/introduction,

As the same time ,Is there a configuration that supports both correct access via https://linkis.apache.org/docs/1.1.1/introduction

**Comment by Josh-Cena** [comment]
You can set up a server redirect? Like redirecting `/docs/latest/*` to `/docs/1.1.1`.

**Reply by slorber** [reply]
We were thinking about using a "versions aliases" feature in the past here: https://github.com/facebook/docusaurus/issues/3285

But it never got implemented.

If we implement redirects, it would be client-side redirects and would not be very great, just a workaround. You'd better use server redirects.

We could render twice the same version/docs using 2 different paths (2 docs would need to have the same canonical URL for SEO reasons). This is not ideal either because it would likely mean increasing build times.

---


Didn't try, but maybe after a build you could try doing a `cp -R website/build/docs/latest website/build/docs/1.1.1`

I'm not sure of the possible side-effects of doing that, but it may work nicely to manually create your own "version aliases"
